src/components/vars.sass

```diff
@@ -323,6 +325,7 @@ $highlight-blue-50-35: #c3efff  // $highlight-blue-50 @ 35% opacity
 
 $header-height: 55px
 $nav-tabs-width: 90px
+$collapsed-tab-width: 38px;
```

Comment:
This is confusing since we mix sass and scss, but in these sass files, you don't need the semi-colon.